Makefile.core.mk

```diff
@@ -37,3 +37,4 @@ gen-check: gen check-clean-repo
 
 presubmit: export RUSTFLAGS = -D warnings
 presubmit: build test lint gen-check build-features
+presubmit: cargo install cargo-audit && cargo audit
```

Comment:
One minor thing - I think we will want to do the CVE checking only in postsubmit. Right now the postsubmit job (confusingly) also runs `make presubmit` but we could make a new target that runs the normal presubmit + CVE check and move the job to run that. Or even a periodic job for the CVE scanning

The reason is we generally try to keep the presubmit hermetic, so external changes (new CVE published) don't suddenly break all PRs